src/longhorn_rpc_protocol.h

```diff
@@ -38,6 +38,7 @@ struct Message {
 enum uint32_t {
 	TypeRead,
 	TypeWrite,
+	TypeUnmap,
```

Comment:
Suggest appending `TypeUnmap` to the last value of this enum, because the order of enum values could matter for backward compatibility somehow.